thirdeye/thirdeye-frontend/app/pods/components/modals/entity-mapping-modal/component.js

```diff
@@ -0,0 +1,450 @@
+/**
```

Comment:
Please review this. (half of it are jsdoc) 

I'll add a todo to abstract the fetching into a service. This is the mvp